ecpp/average.ecpp

```diff
@@ -308,12 +309,30 @@ bool database_ready;
                 // we got some datas?
                 if (zmsg_size(recv_msg) >= 2) {
                     if (!got_timestamps) {
-                        // first row: "type" label + timestamps receipt (index 0)
-                        csv_data.push_back (std::vector <std::string> {"type"});
+                        // first row: "Time (Sec)" label + timestamps receipt (index 0)
+                        csv_data.push_back (std::vector <std::string> {"Time (UTC)"});
                     }
 
                     // add a new row: type_rep label + datas
-                    csv_data.push_back (std::vector <std::string> {type_rep});
+
+                    std::string correctedType(type_rep);
+
+                    if(checked_source == "realpower.default")
```

Comment:
a detail, usually we add a space separator after the if else while for....  ;)
if (...